src/page-weekly-snapshot/FacilityProjectionChart.tsx

```diff
@@ -91,8 +108,9 @@ const FacilityProjectionChart: React.FC<ProjectionProps> = ({
     axes: [
       {
         orient: "bottom",
-        ticks: 5,
-        tickFormat: (value: Date) => formatDate(new Date(value), "MM/dd"),
+        ticks: numXTicks,
```

Comment:
I don't think this option does anything anymore in the presence of `tickValues` so you can just remove it